### ISSUE TITLE
Tag emulation?

### ISSUE THREAD

**Issue by kordeiro** [DFRobot/DFRobot_NFC#1]
Will it be possible to add tag emulation in this library?

**Comment by tmckenna-1** [comment]
I also was curious about utilizing tag emulation. Is tag emulation on this device possible? And does it utilize I2C or UART? Any help would be greatly appreciated.

**Comment by spaniardmaximus** [comment]
Yes, I'd like a piece of that :). Must find an example because the word in the streets is that it could be possible.
